starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+const { render, response } = require('../app');
+const router  = express.Router();
+const Celebrity = require('../models/celebrity')
+
+/* GET home page */
+router.get('/', (req, res, next) => {
+  Celebrity.find()
+  .then(allCelebs =>{
+    console.log("All celebs:", allCelebs)
+    res.render('../views/celebrities/index', {celebs: allCelebs})
+  })
+  .catch(err => console.log('There has been an error: ', err))
+});
+
+router.get('/new', (req,res) => {
+  res.render('../views/celebrities/newCeleb')
+});
+
+router.get('/:id', (req,res,next) => {
+  const {id} = req.params
+  Celebrity.findById(id)
+  .then(specificCeleb => {
+    res.render('../views/celebrities/celebsPage', specificCeleb)
+  })
+  .catch(err => console.log('There has been an error: ', err))
+});
+
+router.post('/:id/delete', (req,res) => {
+  const {id} = req.params
+  Celebrity.findByIdAndDelete(id)
+  .then( () =>{
+    console.log('Celeb is removed')
+    res.redirect('/celebrities')
+  })
+})
+
+router.post('/', (req,res) => {
```

Comment:
it is good practice to use the same url for the get and post requests that handle the same functionality. for example the get "/new" renders the page with the form to create a new celeb, then the post "/new" should get the information to create the celeb.